### PR TITLE
Fetch config script

### PR DESCRIPTION
**Pull request by saliceti**
Script to fetch config and secrets stored in yaml from different sources and output them to a destination with a chosen format

### CODE REVIEW

File: scripts/fetch_secrets/fetch_secrets.rb [hunk 1]
```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env ruby
+require 'getoptlong'
+require 'logger'
+require 'yaml'
+require 'aws-sdk-ssm'
+require 'erb'
+require 'tempfile'
+
+ALLOWED_SOURCES = ['parameter', 'parameter-path', 'yaml-file']
```
Comment (vigneshmsft) [inline scripts/fetch_secrets/fetch_secrets.rb]: is it worth prefixing `aws-ssm` as we'll be introducting KeyVault?

**Review by vigneshmsft — APPROVED** [review]

**Review by sfawcett123 — APPROVED** [review]
Consider naming the process more generic as suggested, so it could be bat_fetch_config

The AWS Parameters could be prefixed with aws- to help differentiate

File: scripts/fetch_secrets/fetch_secrets.rb (hunk 1 above)
Comment (saliceti) [reply scripts/fetch_secrets/fetch_secrets.rb]: Changed, thanks

**Comment by saliceti** [comment]
> Consider naming the process more generic as suggested, so it could be bat_fetch_config
> 
> The AWS Parameters could be prefixed with aws- to help differentiate

Renamed to fetch_config.rb and prefixed with ssm-

**Pull request merged**